### ISSUE TITLE
Hue: dynamic icons for dimmer items on color channel not working

### ISSUE THREAD

**Issue by metbril** [eclipse/smarthome#3136]
Usually, you need to define the dimmer for a Hue bulb through the color channel.
If you use a slider in the sitemap (with the slider icon), the state for that dimmer item is the RGB code, and that state is used for generating the dynamic item.
The effect is, that dynamic icons in the sitemap will not work.

Example src for the dynamic image:

`http://raspberrypi.local:8080/icon/slider?state=74,78,44&format=svg`

**Comment by rtvb** [comment]
Relation with #866?

**Comment by triller-telekom** [comment]
Hello,

I just tried to reproduce your problem and I used the following setup:

demo.things:
```
Bridge hue:bridge:1 [ ipAddress="192.168.0.64" ] {
	0210 bulb1 [ lightId="1" ]
}
```

demo.items:
```
Dimmer  Light1_Dimmer		{ channel="hue:0210:1:bulb1:color" }
```

demo.sitemap:
```
sitemap demo label="Main Menu"
{
	Frame {
		Slider item=		Light1_Dimmer
	}
}
```

My `basicui` fetches the icon for the slider from this URL:
http://localhost:8080/icon/slider?state=10&format=png

This is working fine and if I adjust the slider, the icon is updated properly. Also in my `classicui` the icon is loaded from this URL:
http://localhost:8080/icon/slider?format=png&state=10

and this is also working as expected.

So which UI are you using? And which version of Eclipse SmartHome/openHAB are you using?
